storage/postgres/tree_storage.go

```diff
@@ -15,11 +15,52 @@
 package postgres
 
 import (
+	"context"
 	"database/sql"
+	"encoding/base64"
+	"fmt"
+	"runtime/debug"
+	"strings"
+	"sync"
 
 	"github.com/golang/glog"
+	"github.com/golang/protobuf/proto"
+	"github.com/google/trillian"
+	"github.com/google/trillian/storage"
+	"github.com/google/trillian/storage/cache"
+	"github.com/google/trillian/storage/storagepb"
 )
 
+const (
+	placeholderSQL        = "<placeholder>"
+	insertSubtreeMultiSQL = `INSERT INTO subtree(tree_id, subtree_id, nodes, subtree_revision) ` + placeholderSQL
+	selectSubtreeSQL      = `
+ SELECT x.subtree_id, x.max_revision, subtree.nodes
```

Comment:
Can probably indent this all to help with readability.

Reply:
Should be fine but it does seem to indent the actual string https://play.golang.org/p/HRgEaPIx4J9

Reply:
Yup, but I don't think it'll cause any problems and we win a bit on tidy source.